package.json

```diff
@@ -75,6 +77,7 @@
     "git-branch": "^2.0.1",
     "googleapis": "^85.0.0",
     "mime-types": "^2.1.32",
+    "postcss": "^7.0.36",
```

Comment:
Do we use postcss directly? I'm sure it is already added somewhere by `@vue/cli` or some other dependency (because even though you've set it here, no changes were made in `package-lock.json`)

If it's not something we use, I'd prefer avoiding listing it here (doesn't make sense IMHO)